dask_cloudprovider/providers/azure/azureml.py

```diff
@@ -514,23 +655,27 @@ async def __setup_port_forwarding(self):
                     f"setsid socat tcp-listen:{self.port[1]},reuseaddr,fork tcp:{scheduler_ip}:{port[0]} &"
                 )
         else:
+            forwarding_option = "L"
```

Comment:
What is the purpose of moving this out into a variable?

Reply:
Thanks, Jacob. No specific purpose. I was testing remote forwarding with -R. I can change it back if this is a concern.

Reply:
Yeah, I would prefer to reduce complexity as much as possible. It just makes the code a little harder to parse when coming back to it.

Reply:
Updated. Thanks.